### ISSUE TITLE
Rolling ontology build should have a time stamp

### ISSUE THREAD

**Issue by pgaudet** [geneontology/pipeline#384]
Hello, 

The header of the .obo file of the Rolling ontology build (and maybe other files, I didn't check all) should include not just the date, but also a timestamp, since it's built multiple times a day, we have no way of knowing when was the last successful one. 

Thanks, Pascale

**Comment by kltm** [comment]
@pgaudet I think, for our purposes, this would functionally be resolved by https://github.com/geneontology/pipeline/issues/382

**Comment by pgaudet** [comment]
But I thought you wanted to keep multiple time.a day ontology generation, in case anything breaks?

**Comment by kltm** [comment]
Yes, but these are separate use cases (in my formulation). We could blend them, but I think it may be prudent to keep them separate: QC and consumption.
